figma-shop/src/app/pages/about/about.component.spec.ts

```diff
@@ -1,20 +1,20 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Remove all specs from your project - we will not write any unit tests.
Generate components with skipping them: `ng g c component-name --skip-tests`

Reply:
All specs have been deleted